src/sonic-yang-models/tests/yang_model_tests/tests_config/feature.json

```diff
@@ -113,7 +121,8 @@
                         "has_timer": "FALSE",
                         "has_global_scope": "TRUE",
                         "has_per_asic_scope": "TRUE",
-                        "check_up_status": "FALSE"
+                        "check_up_status": "FALSE",
+                        "support_syslog_rate_limit": "TRUE"
```

Comment:
Is there any reason this one is uppercase?

Reply:
This is a negative test